src/store/conversationsStore.js

```diff
@@ -121,6 +121,15 @@ const mutations = {
 		Vue.set(state.conversations[token], 'lastMessage', lastMessage)
 	},
 
+	updateUnreadMessages(state, { token, unreadMessages, unreadMention }) {
```

Comment:
this will be wiped after 30 seconds by the room list update?

Reply:
Yes, exactly. It's just a shortcut to make it updater faster.

in my test I waited for 30 seconds to verify that the values were the same

Reply:
Okay seems to work somehow. We are not marking the chat read anymore when not open I guess.

Reply:
yes, this is new behavior on master since the introduction of the unread marker